modules/gdnative/net/multiplayer_peer_gdnative.cpp

```diff
@@ -68,7 +68,7 @@ void MultiplayerPeerGDNative::set_transfer_mode(TransferMode p_mode) {
 }
 
 NetworkedMultiplayerPeer::TransferMode MultiplayerPeerGDNative::get_transfer_mode() const {
-	ERR_FAIL_COND_V(interface == NULL, TRANSFER_MODE_UNRELIABLE);
+	if (interface == NULL) return TRANSFER_MODE_UNRELIABLE;
```

Comment:
CC @Faless

Reply:
To be honest I don't like removing the ERR_FAILs much, could we add an optional `DEFVAL` override to `ADD_PROPERTY` (`ClassDB::add_property`), which populate the `default_values` used in `ClassDB::class_get_default_property_value`? Would that impact memory or do we load default values anyway? @bojidar-bg 
Otherwise really nice work, it's going to be a great feature! :heart:

Reply:
I guess having a way to specify the default value in `ADD_PROPERTY` would be pretty cool indeed, as it would allow me to not remove the checks. Will look into it..